src/test/java/org/folio/rest/impl/GOBIIntegrationServiceResourceImplTest.java

```diff
@@ -541,6 +542,21 @@ private void handleGetLocation(RoutingContext ctx) {
         .putHeader(HttpHeaders.CONTENT_TYPE, APPLICATION_JSON)
         .end(locations.encodePrettily());
     }
+
+    private void handleGetConfigurationsEntries(RoutingContext ctx) {
+
+      logger.info("got configurations entries request: " + ctx.request().query());
+
+      JsonObject configurationsEntries = new JsonObject()
```

Comment:
So we're only mocking the "no configuration" exists path here?

Reply:
Currently, yes. I hooked up the code to retrieve the config data, which was not hooked up before. However, if the path isn't there in the mock server, then the API tests fail. We'll need to figure out what mappings we want to return here or if we want to return any mappings for the API tests.